exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
```

Comment:
given that this claims to follow x-common 1.0.0, now I am obligated to ask: is the omission of https://github.com/exercism/problem-specifications/blob/master/exercises/rna-transcription/canonical-data.json#L48-L51 intentional? Please explain why if so?